src/main/java/org/sagebionetworks/web/server/servlet/filter/BotHtml.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.web.server.servlet.filter;
```

Comment:
Small helper class to organize html that should be included in pages sent back to bot traffic.  This is generally expensive to calculate, so we don't include this for normal visitors.  It typically includes a lot of extra links and basic information, rather than the skeleton page that the client fills in (usually with the same or similar information)